aidermacs-backend-vterm.el

```diff
@@ -45,6 +45,9 @@
 (declare-function aidermacs--is-aidermacs-buffer-p "aidermacs")
 (declare-function aidermacs-get-buffer-name "aidermacs")
 
+;; useful because we want to override "RET" key for evil mode insert state
+(declare-function evil-define-minor-mode-key "evil-core")
```

Comment:
Do we need this in comint as well?

Reply:
I don’t think evil is need in aidermacs-comint-backend, because I don’t think you need to rebind RET for comint. Since I think you didn’t setup a dedicated RET key binding for comint aider buffer.